daemon/daemon_main.go

```diff
@@ -1007,6 +1007,7 @@ func initEnv(cmd *cobra.Command) {
 
 	option.Config.Opts.SetBool(option.Debug, option.Config.Debug)
 	option.Config.Opts.SetBool(option.DebugLB, option.Config.Debug)
+	option.Config.Opts.SetBool(option.DebugPolicy, option.Config.Debug)
```

Comment:
This is not that useful as running in full debug mode won't be feasible in production and we will want to selectively run this in prod.